learning-ojs/en/journal-setup.md

```diff
@@ -76,11 +76,11 @@ From here, you can change the name or abbreviation of the section.
 
 **Word Count**: Use this to limit the number of words for abstracts in this section.
 
-**Review Form**: You can select specific review forms for each section. Learn more about Review Forms here.
+**Review Form**: You can select specific review forms for each section. Learn more about Review Forms [here](./editorial-workflow.md#review).
```

Comment:
For accessibility, links should not be wrapped around text that provide no context. Can you change the link from:

> Learn more about Review Forms [here](...)

To:

> [Learn more about Review Forms](...)